analyzers/tests/SonarAnalyzer.UnitTest/Rules/ForeachLoopExplicitConversionTest.cs

```diff
@@ -25,30 +25,34 @@ namespace SonarAnalyzer.UnitTest.Rules
     [TestClass]
     public class ForeachLoopExplicitConversionTest
     {
+        private readonly VerifierBuilder verifier = new VerifierBuilder<ForeachLoopExplicitConversion>();
+
         [TestMethod]
         public void ForeachLoopExplicitConversion() =>
-            OldVerifier.VerifyAnalyzer(@"TestCases\ForeachLoopExplicitConversion.cs", new ForeachLoopExplicitConversion());
+            verifier.AddPaths("ForeachLoopExplicitConversion.cs").Verify();
 
         [TestMethod]
         public void ForeachLoopExplicitConversion_CSharp10() =>
-            OldVerifier.VerifyAnalyzerFromCSharp10Library(@"TestCases\ForeachLoopExplicitConversion.CSharp10.cs", new ForeachLoopExplicitConversion());
+            verifier.AddPaths("ForeachLoopExplicitConversion.CSharp10.cs")
+                .WithConcurrentAnalysis(false)
+                .WithOptions(ParseOptionsHelper.FromCSharp10)
+                .Verify();
 
         [TestMethod]
         public void ForeachLoopExplicitConversion_CodeFix() =>
-            OldVerifier.VerifyCodeFix<ForeachLoopExplicitConversionCodeFix>(
-                @"TestCases\ForeachLoopExplicitConversion.cs",
-                @"TestCases\ForeachLoopExplicitConversion.Fixed.cs",
-                new ForeachLoopExplicitConversion());
-
+            verifier.WithCodeFix<ForeachLoopExplicitConversionCodeFix>()
+                .AddPaths("ForeachLoopExplicitConversion.cs")
+                .WithCodeFixedPaths("ForeachLoopExplicitConversion.Fixed.cs")
+                .VerifyCodeFix();
 #if NET
```

Comment:
Also, I am not sure it is documented but at some point we agreed to have an empty line before and after each preprocessor directive.